modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/document/library/DocumentLibraryDDMFormFieldTemplateContextContributorTest.java

```diff
@@ -258,18 +260,24 @@ public void testGetParametersShouldContainFileEntryURL()
 		ddmFormFieldRenderingContext.setProperty(
 			"ddmFormInstanceRecordId", RandomTestUtil.randomLong());
 
-		Map<String, Object> parameters =
-			documentLibraryDDMFormFieldTemplateContextContributor.getParameters(
-				new DDMFormField("field", "document_library"),
-				ddmFormFieldRenderingContext);
+		HttpServletRequest httpServletRequest =
+			ddmFormFieldRenderingContext.getHttpServletRequest();
+
+		RequestBackedPortletURLFactory requestBackedPortletURLFactory =
+			RequestBackedPortletURLFactoryUtil.create(httpServletRequest);
 
 		Assert.assertEquals(
 			String.valueOf(new TestMockLiferayPortletURL()),
-			parameters.get("fileEntryURL"));
+			PortletURLBuilder.create(
+				requestBackedPortletURLFactory.createActionURL(
+					GetterUtil.getString(
+						_portal.getPortletId(httpServletRequest),
+						DDMPortletKeys.DYNAMIC_DATA_MAPPING_FORM))
+			).buildString());
```

Comment:
@icgsbr  the purpose of this assert is to check if the `getParameters` method is returning the expected value for `fileEntryURL`. So we should use `parameters.get("fileEntryURL")` on the actual value.

After your changes, you are forcing the actual value to be what we expect, so unfortunately I cannot accept this change.